test/bigtest/tests/utils-test.js

```diff
@@ -172,4 +173,25 @@ describe('utils', () => {
       expect(convertToSlipData(pickSlipsWithEmptyDate, intl, timeZone, locale)).to.deep.equal(expectSlipDataWithEmptyDate);
     });
   });
+
+  describe('Format Note referrer entity data', () => {
+    it('should return formatted referrer entity data', () => {
+      const data = {
+        entityName: 'name',
+        entityType: 'type',
+        entityId: 'id',
+      };
+      const expectedData = {
+        name: data.entityName,
+        type: data.entityType,
+        id: data.entityId,
+      };
+
+      expect(formatNoteReferrerEntityData(data)).to.deep.equal(expectedData);
+    });
+
+    it('should return false', () => {
```

Comment:
Could you please extend the description for this case so we reason about it without looking at the actual code.
```suggestion
    it('should return false', () => {
```